app/models/ckb_sync/node_data_processor.rb

```diff
@@ -121,7 +121,8 @@ def generate_address_in_advance(cellbase)
 
     def build_block(node_block)
       header = node_block.header
-      epoch_info = CkbUtils.get_epoch_info(header.epoch)
+      epoch = "0x#{CKB::Utils.to_hex(header.epoch).split(//).last(6).join("")}".hex
```

Comment:
Why epoch use last 6 characters?

Reply:
https://github.com/nervosnetwork/ckb/pull/1585

Reply:
The lowest 24 bits represent the current epoch number